docs/install-azure-repos.md

```diff
@@ -50,6 +48,13 @@ To install Frogbot on Azure Repos repositories, follow these steps.
          - task: CmdLine@2
            displayName: 'Download and Run Frogbot'
            env:
+              # [Mandatory if the two conditions below are met]
+              # 1. The project uses npm, yarn 2, NuGet or .NET to download its dependencies
+              # 2. The `installCommand` variable isn't set in your frogbot-config.yml file.
+              #
+              # The command that installs the project dependencies (e.g "npm i", "nuget restore" or "dotnet restore")
+              JF_INSTALL_DEPS_CMD: ""
+   
```

Comment:
In case the `frogbot-config.yml` file isn't used, we should also add the `JF_GIT_REPO` environment variable.
```
 # [Mandatory]
 # Azure Repos repository name
   JF_GIT_REPO: ""
```

Reply:
Good catch!
Since we need to find the default value for JF_GIT_REPO in Azure DevOps and JFrog Pipelines, I'll leave it to you @omerzi to investigate and test, and add this in a follow-up PR.